packages/widgetbook/lib/src/widgetbook.dart

```diff
@@ -37,12 +37,12 @@ import 'package:widgetbook_models/widgetbook_models.dart';
 class Widgetbook<CustomTheme> extends StatefulWidget {
   const Widgetbook({
     super.key,
-    this.children = const <MultiChildNavigationNodeData>[],
+    this.directories = const <MultiChildNavigationNodeData>[],
     required this.appInfo,
```

Comment:
Thought I can remove this in a separate PR, maybe replace it for an optional `appTitle` to populate the title of the `MaterialApp`?